src/app/forms/templates/test-records/master.template.ts

```diff
@@ -475,6 +476,15 @@ export const masterTpl: Record<VehicleTypes, Partial<Record<keyof typeof TEST_TY
       notes: NotesSection,
       reasonForCreation: reasonForCreationSection
     },
+    testTypesDeskBasedGroup3: {
```

Comment:
It's also in the LGV section for the master template, and this test group is for Small Trailers
I forget if we confirmed what we were going to do about light trailers or not, but I don't think it should be in this part? (correct me if I'm wrong)

Reply:
Good point man, after reading the ticket carefully, it looks like this PR is blocked. The Light TRL vehicle type needs to be implemented first for this to work. I will bring it in tomorrow's stand up.

Reply:
This now sits under `small trl`.